src/less/wizard-stepper/base/wizard-stepper.less

```diff
@@ -18,247 +34,184 @@
-    }
+.wizard-stepper__item--inprogress {
+    .highlight-upcoming(@wizard-step-upcoming-color);
+
+    color: @wizard-step-confirmation-color;
 }
 
-.wizard-stepper__item--transition-default .wizard-stepper__icon::after {
-    background-color: @wizard-step-default-color !important;
+// Place transitions at end to take higher priority
+.wizard-stepper__item--transition-error {
+    .highlight-upcoming(@wizard-step-error-color);
 }
 
-.wizard-stepper__item--transition-upcoming .wizard-stepper__icon::after {
-    background-color: @wizard-step-upcoming-color !important;
+.wizard-stepper__item--transition-insufficient {
+    .highlight-upcoming(@wizard-step-insufficient-color);
 }
 
-.wizard-stepper__item--transition-inprogress .wizard-stepper__icon::after {
-    background-color: @wizard-step-confirmation-color !important;
+.wizard-stepper__item--transition-default {
+    .highlight-upcoming(@wizard-step-default-color);
 }
 
-.wizard-stepper__item--transition-confirmation .wizard-stepper__icon::after {
-    background-color: @wizard-step-confirmation-color !important;
+.wizard-stepper__text {
+    margin-top: 8px;
+    text-align: center;
 }
 
-.wizard-stepper__item--transition-insufficient .wizard-stepper__icon::after {
-    background-color: @wizard-step-insufficient-color !important;
+.wizard-stepper__item--inprogress .wizard-stepper__text,
+.wizard-stepper__item--default .wizard-stepper__text {
+    color: @wizard-step-default-color;
 }
 
-.wizard-stepper__item--transition-error .wizard-stepper__icon::after {
-    background-color: @wizard-step-error-color !important;
+.wizard-stepper__item--inprogress .wizard-stepper__text,
+.wizard-stepper__item--confirmation .wizard-stepper__text.wizard-stepper__text--bold {
+    font-weight: bold;
 }
 
 .wizard-stepper__item {
+    display: flex;
+    flex-direction: column;
     margin: 0 auto;
-    min-width: 80px;
+    margin-bottom: 25px;
     position: relative;
     text-align: center;
+}
 
-    .wizard-stepper__text {
-        margin-top: 8px;
-    }
+.wizard-stepper__item:first-child {
+    margin-left: 0;
+}
 
-    &--default {
-        color: @wizard-step-default-color;
+.wizard-stepper__item:first-child .wizard-stepper__icon::before {
+    height: 0;
+}
 
-        .wizard-stepper__icon::before,
-        .wizard-stepper__icon::after {
-            background-color: @wizard-step-default-color;
-        }
+.wizard-stepper__item:last-child {
+    margin-right: 0;
+}
 
-        .wizard-stepper__text {
-            color: @wizard-step-default-color;
-        }
-    }
+.wizard-stepper__item:last-child .wizard-stepper__icon::after {
+    height: 0;
+}
 
-    &--upcoming {
-        color: @wizard-step-upcoming-color;
+.wizard-stepper {
+    margin: 16px 0;
+    overflow: auto;
+}
 
-        .wizard-stepper__icon::before,
-        .wizard-stepper__icon::after {
-            background-color: @wizard-step-upcoming-color;
-        }
-    }
+.wizard-stepper.wizard-stepper--vertical {
```

Comment:
@aglig Nitpick, but is the chained class still needed?

Reply:
Will remove, your right not needed.
Also will remove the nested styles